packages/icons/icons.json

```
@@ -933,217 +933,217 @@
         "codepoint": 62050
     },
     {
         "displayName": "Zoom to fit",
         "iconName": "zoom-to-fit",
-        "tags": "fit, scale, resize, adjust",
+        "tags": "fit, scale, resize, adjust, expand, zoom out, snapshot",
         "group": "action",
         "codepoint": 62240
     },
     {
         "displayName": "Add to artifact",
         "iconName": "add-to-artifact",
-        "tags": "list, plus",
+        "tags": "list, plus, text, add, create, add item, add to list, menu, more, hamburger, add section, add page",
         "group": "action",
         "codepoint": 61703
     },
     {
         "displayName": "Map marker",
         "iconName": "map-marker",
-        "tags": "pin, map, location, position, geography, world",
+        "tags": "pin, map, location, position, geography, world, point",
         "group": "interface",
         "codepoint": 62010
     },
     {
         "displayName": "Chart",
         "iconName": "chart",
-        "tags": "arrow, increase, up, line, bar, graph",
+        "tags": "arrow, increase, up, line, bar, graph, trend, growth, ?, stonks, increasing trend, trendline, upward",
         "group": "data",
         "codepoint": 61767
     },
     {
         "displayName": "Control",
         "iconName": "control",
-        "tags": "squares, layout",
+        "tags": "squares, layout, filled layout, filled module, module, page, dashboard, workshop, modular, application, window, app, tile, tessellate, task view",
         "group": "interface",
         "codepoint": 61799
     },
     {
         "displayName": "Multi select",
         "iconName": "multi-select",
-        "tags": "layers, selection",
+        "tags": "layers, selection, list, files, multiple windows, tabs",
         "group": "interface",
         "codepoint": 62030
     },
     {
         "displayName": "Direction: left",
         "iconName": "direction-left",
-        "tags": "pointer, west",
+        "tags": "pointer, west, direction, space, arrow left, back, pointer left",
         "group": "interface",
         "codepoint": 61819
     },
     {
         "displayName": "Direction: right",
         "iconName": "direction-right",
-        "tags": "pointer, east",
+        "tags": "pointer, east, arrow, bullet, tag, send, location",
         "group": "interface",
         "codepoint": 61820
     },
     {
         "displayName": "Database",
         "iconName": "database",
-        "tags": "stack, storage",
+        "tags": "stack, storage, can, data, source, dataset, data source",
         "group": "data",
         "codepoint": 61812
     },
     {
         "displayName": "Pie chart",
         "iconName": "pie-chart",
-        "tags": "circle, part, section",
+        "tags": "circle, part, section, chart, visualization",
         "group": "data",
         "codepoint": 62068
     },
     {
         "displayName": "Full circle",
         "iconName": "full-circle",
-        "tags": "dot, point",
+        "tags": "dot, point, orb, black",
```

```suggestion
        "tags": "dot, point, orb, filled",
```

icons may have any color, and they are white(-ish) in dark theme